docker-compose.yaml

```diff
@@ -2,19 +2,138 @@ version: "3.9"
 
 services:
   cardano-node:
-    image: inputoutput/cardano-node:1.35.4
+    image: inputoutput/cardano-node:latest
```

Comment:
I am not sure if use latest is good, cuz `cardano-node` might break, while I guess we want need to change version often.

Reply:
```suggestion
    image: inputoutput/cardano-node:1.35.4
```